<ui version="4.0" >
 <class>jewelryLoginClass</class>
 <widget class="QWidget" name="jewelryLoginClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>387</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>387</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>jewelryLogin</string>
  </property>
  <widget class="QComboBox" name="cbName" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>38</y>
     <width>173</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>173</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>173</width>
     <height>32</height>
    </size>
   </property>
   <property name="font" >
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>93</y>
     <width>173</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>173</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>173</width>
     <height>32</height>
    </size>
   </property>
   <property name="font" >
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="logMinClose" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>10</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="logClose" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>140</y>
     <width>77</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>77</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>77</width>
     <height>30</height>
    </size>
   </property>
   <property name="font" >
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="logEnter" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>140</y>
     <width>77</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>77</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>77</width>
     <height>30</height>
    </size>
   </property>
   <property name="font" >
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoDefault" >
    <bool>true</bool>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="jewelrysystem_jj.qrc" />
 </resources>
 <connections/>
</ui>
